### PR TITLE
Add a line break before kill

### PR DESCRIPTION
**Pull request by jolopezl**
A fix for https://github.com/compiler-explorer/compiler-explorer/issues/5233

### CODE REVIEW

File: lib/exec.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ export function executeDirect(
             timedOut = true;
             kill();
             streams.stderr += '\nKilled - processing time exceeded';
+            streams.stderr += '\n';
```
Comment (mattgodbolt) [inline lib/exec.ts:117]: Please add this to the previous line, likew you did in the `exec-tests.ts`
```suggestion
            streams.stderr += '\nKilled - processing time exceeded\n';
```

**Review by mattgodbolt — COMMENTED** [review]
Thanks for the contribution - can you make the tiny nitpick change I suggested, then we're good to go :)

**Comment by jeremy-rifkin** [comment]
Thanks!

**Comment by jolopezl** [comment]
You are welcome! I just added the linebreak as suggested.

**Review by jeremy-rifkin — APPROVED** [review]
Looks good to me, thank you again :)

**Comment by jeremy-rifkin** [comment]
Ope, sorry, why are tests failing?

**Comment by jeremy-rifkin** [comment]
Looks like there's one other place in the test file where the expected output needs to be updated. After that, looks good to me :)

**Comment by partouf** [comment]
now live

**Pull request merged**